packages/web/components/stake/stake-onboarding.tsx

```diff
@@ -94,11 +97,16 @@ export const StakeOnboarding: React.FC<StakeOnboardingProps> = ({
 
   const { stakingAPR } = useGetApr();
 
+  const handleClose = () => {
+    () => setShowStakeIntroModal(false);
```

Comment:
Is that arrow function required? Can't we just call `setShowStakeIntroModal(false)`?

Reply:
yes. copypasta mistake

Reply:
@DavideSegullo updated